subcommand/health-sync/command.go

```diff
@@ -181,6 +190,27 @@ func (c *Command) setChecksCritical(consulClient *api.Client, taskID string, ser
 	return result
 }
 
+func (c *Command) logout(tokenFile string) error {
+	cfg := api.DefaultConfig()
+	if c.config.ConsulHTTPAddr != "" {
+		cfg.Address = c.config.ConsulHTTPAddr
+	}
+	if c.config.ConsulCACertFile != "" {
+		cfg.TLSConfig.CAFile = c.config.ConsulCACertFile
+	}
+	cfg.TokenFile = tokenFile
```

Comment:
suggestion: instead of doing the `filepath.Join` in both calls to `logout` from `realRun`, you could move that join step in here (since it's using `c.config.BootstrapDir`) to make the `logout` calls a bit more "readable"